powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/model/request/RemoveActivationRequest.java

```diff
@@ -28,8 +30,14 @@
 @Data
 public class RemoveActivationRequest {
 
+    @Schema(description = "Activation identifier")
+    @NotBlank(message = "Activation ID must not be empty when blocking activation")
```

Comment:
Wrong usage in description
```suggestion
    @NotBlank(message = "Activation ID must not be empty when removing activation")
```